modules/drone_odometry_local.py

```diff
@@ -107,14 +112,22 @@ def create(cls,
 
         return True, DroneOdometryLocal(cls.__create_key, position, orientation)
 
-    def __init__(self,
-                 class_private_create_key,
-                 position: DronePositionLocal,
-                 orientation: DroneOrientationLocal):
+    def __init__(
+        self,
+        class_private_create_key: object,
+        position: DronePositionLocal,
+        orientation: DroneOrientationLocal,
+    ) -> None:
         """
         Private constructor, use create() method.
         """
         assert class_private_create_key is DroneOdometryLocal.__create_key, "Use create() method"
 
         self.position = position
         self.orientation = orientation
+
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        return f"DroneOdometryLocal: {self.position}, {self.orientation}"
```

Comment:
Keep __str__() function consistent with other modules. Either just return the string or always assign representation = f"...." and return representation